demand_water.md

```diff
@@ -0,0 +1,168 @@
+
+Municipal water consumption is also modeled; the distinction between the two comes from an assumed overall municipal water supply efficiency, based on [Shiklomanov 2000](#shiklomanov2000). Where the withdrawal volume indicates the total water input to the municipal water supply system, consumption indicates only the water that is used by consumers in a way that it is not returned to the immediate water environment.
+
+In the future, the ratio between withdrawals and consumption is assumed static, so both the withdrawals and consumption grow according to the equation above. Future work on adaptive measures can explore the potential changes to withdrawl and consumption intensities.
+
+For more information, see  [Hejazi et al. 2013](#hejazi2013) and summarized in [Hejazi et al. 2014](#hejazi2014).
+
+### Types of water in GCAM
+
+* **water withdrawals**: water diverted or withdrawn from a surface water or groundwater source ([Vickers 2001](#vickers2001)).
+* **water consumption**: water use that permanently withdraws water from its source; water that is no longer available because it has evaporated, been transpired by plants, incorporated into products or crops, consumed by people or livestock, or otherwise removed from the immediate water environment ([Vickers 2001](#vickers2001)).
+* **biophysical water consumption**: total water required for crop evapo-transpiration; the sum of "blue" and "green" water in [Mekonnen and Hoekstra 2011](#mekonnen2011)
+* **seawater**: water from the oceans, including brackish estuaries, that is withdrawn for cooling thermo-electric power plants, or used in primary energy production.
+
+
+### Basin-To-Region and Basin-To-Sector Mapping
+
+Agricultural water demand is modeled at the basin level directly. Demands for water from other sectors are modeled at the geopolitical region and then mapped to the basin. For more information, see the detailed description of the [basin mapping](details_#basin-to-regionandbasin-to-sectormapping).
+
+
+## Equations 
+The equations that determine water demand are described here.
+
+#### Municipal water demand
+
+The equation is shown below for per-capita municipal water demands (*pcW*) from time period *t-1* to time period *t*.
+
+$$
+pcW_t = pcW_{t-1} * (\frac{pcGDP_t}{pcGDP_{t-1}})^{0.37} *
+  (\frac{P_t}{P_{t-1}})^{-0.33} * (1 - Tech_t)
+$$
+
+where pcGDP is per-capita GDP, P is water price, and Tech stands for autonomous technological change, an exogenous assumption.
+
+See `calcDemand` in [minicam_price_elasticity_function.cpp](https://github.com/JGCRI/gcam-core/blob/master/cvs/objects/functions/source/minicam_price_elasticity_function.cpp).
+
+## Policy options 
```

Comment:
I would recommend deleting the policy section. We have not done much in the way of policies with respect to water. The only thing I can see going here eventually would be the climate response to water supply, but I do not believe that is in the scope here.